lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -249,7 +249,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_belarus
 
-      outcome :outcome_os_italy
+      outcome :outcome_opposite_sex_marriage_italy_residing_in_uk_or_italy
```

Comment:
I've consistently named the outcomes using `residing_in_uk_or_<ceremony-country>` in PR #2409.